src/DistributedComputations/distributed_fft_based_poisson_solver.jl

```diff
@@ -32,21 +28,17 @@ Return an FFT-based solver for the Poisson equation,
 ∇²φ = b
 ```
 
-on `Distributed`itectures.
+for `Distributed` architectures.
 
 Supported configurations
 ========================
 
-We support two "modes":
-
-  1. Vertical pencil decompositions: two-dimensional decompositions in ``(x, y)``
-     for three dimensional problems that satisfy either `Nz > Rx` or `Nz > Ry`.
+1. Three dimensional configurations with two-dimensional decompositions in ``(x, y)`` 
+where `Ny ≥ Rx` and `Ny % Rx = 0`, and `Nz ≥ Ry` and `Nz % Ry = 0`.
 
-  2. One-dimensional decompositions in either ``x`` or ``y``.
-
-Above, `Nz = size(global_grid, 3)` and `Rx, Ry, Rz = architecture(local_grid).ranks`.
-
-Other configurations that are decomposed in ``(x, y)`` but have too few `Nz`,
+2. Two dimensional configurations decomposed in ``x`` where `Ny ≥ Rx` and `Ny % Rx = 0`
+    
+Other configurations that are decomposed in ``(x, y)``,
 or any configuration decomposed in ``z``, are _not_ supported.
 
 Algorithm for two-dimensional decompositions
```

Comment:
This is a minor point, and I understand this isn't language introduced by this PR, but for me calling a decomposition in two dimensions a "2D decomposition" can be a little confusing. It perhaps can imply that it's a decomposition for 2D domains (I see that you clarify below that it's not). What are your thoughts about avoiding the word "2D" and calling it "x-y decomposition". Or maybe pencil decomposition, which seems to me to be a more common term?

Again, minor point, so feel free to ignore it.

Reply:
I agree, I have changed all the wording to `pencil` and `slab` decomposition as well as added an intro to describe the language used.